PsychSourceGL/Source/Common/Base/PsychAuthors.c

```diff
@@ -212,6 +212,33 @@ void InitPsychAuthorList(void)
         "nuha@sr-research.com",
         "http://sr-research.com"
     );
+    PsychAddAuthor(
+        // Eyelink toolbox:
+        "Brian",
+        "",
+        "Richardson",
+        "br",
+        "brian@sr-research.com",
+        "https://sr-research.com"
+    );
+    PsychAddAuthor(
```

Comment:
This entry for you "Brian Richardson" should be removed. Reason is that I already added your author record, as mentioned previously, so this already exists in the most recently released PTB 3.0.19.4. Adding it here again will add the record twice and cause the Eyelink mex file to abort with a critical error due to redundant author records.

The records of your other colleagues are a good addition though.